assets/src/blocks/ENForm/ENFormSettings.js

```diff
@@ -184,26 +190,26 @@ const getPageListByType = () => {
   }
 
   let flattenedPages = [];
-  for (var i in pages) {
-    let pagesByType = pages[i].map(page => {
-      return { label: page.name, value: page.id };
+  for (const i in pages) {
```

Comment:
Here as well I believe it should be `let` instead of `const` 🤔

Reply:
Same thing too as above, `for-in-loop` re-initialises the variable for each iteration